pwiz_tools/Skyline/Model/Results/IonMobilityFinder.cs

```diff
@@ -103,6 +103,17 @@ public bool UseHighEnergyOffset
             if (_totalSteps == 0)
                 return measured;
 
+            // Before we do anything else, make sure the raw files are present
+            foreach (var f in fileInfos)
+            {
+                if (!ScanProvider.FileExists(_documentFilePath, f.FilePath))
+                {
+                    throw new FileNotFoundException(TextUtil.LineSeparate(Resources.IonMobilityFinder_ProcessMSLevel_Failed_using_results_to_populate_ion_mobility_library_,
+                    string.Format(Resources.ScanProvider_GetScans_The_data_file__0__could_not_be_found__either_at_its_original_location_or_in_the_document_or_document_parent_folder_,

```

Comment:
Something is wrong with the indentation here.

Reply:
fixed, thanks